examples/multiple.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Alma Widgets multiple configuration example</title>
+
+    <link rel="stylesheet" href="../dist/widgets.umd.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <!-- <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@alma/widgets@2.5.1/dist/widgets.min.css"
+    /> -->
+  </head>
+  <body>
+    <div class="body">
+      <div>
+        <img src="../src/assets/alma.svg" class="alma-logo" />
+      </div>
+      <div class="menu">
+        <a href="./basic.html">Basic page</a>
+      </div>
+      <div class="main">
+        <div class="image">
+          <img src="./sac-a-main.jpeg" />
+        </div>
+        <div>
+          <div class="product">
+            <h1>Sac à main</h1>
+            <select onchange="generateRenderer()" id="quantity">
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+              <option value="10">10</option>
+            </select>
+            <h3>450 €</h3>
+            <p>Deluxe edition, bigger price, same product</p>
+            <div id="alma-widget-payment-plans"></div>
+            <h3>250 €</h3>
+            <p>Basic edition, lower price, smarter purchase</p>
+
+            <div id="alma-widget-payment-plans2"></div>
+            <div id="alma-widget-modal"></div>
+            <div class="add-to-cart">
+              <button>Ajouter au panier</button>
+            </div>
+          </div>
+          <div class="open-modal-ctas">
+            <a href="#" id="alma-open-modal-button">Open modal with clickableSelector</a>
+            <a href="#" onClick="openModal()">Open modal with open function</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <script src="../dist/widgets.umd.js"></script>
+    <!-- <script src="https://cdn.jsdelivr.net/npm/@alma/widgets@2.5.1/dist/widgets.umd.js"></script> -->
```

Comment:
```suggestion
```

Reply:
I actually want this line to be available as we need to often switch between the two. This allow a comment/uncomment really fast.

Also our clients use the cdn so to test a specific version it's easier.